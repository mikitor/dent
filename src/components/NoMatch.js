import React from 'react';
import { Link } from 'react-router-dom';

const NoMatch = () => (
  <div>
    <h3>Page not found</h3>
    <Link to="/">Home</Link>
  </div>
);

export default NoMatch;
