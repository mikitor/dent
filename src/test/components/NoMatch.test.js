import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from '../../components/NoMatch';

it('should render NoMatch component', () => {
  const wrapper = shallow(<NoMatch />);
  expect(wrapper).toMatchSnapshot();
});
