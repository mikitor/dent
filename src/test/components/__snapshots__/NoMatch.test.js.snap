// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NoMatch component 1`] = `
<div>
  <h3>
    Page not found
  </h3>
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
